scripts/parser.js

```diff
@@ -63,9 +63,38 @@ async function getJobsinDay(){
     await mbo.class.classes({
         "StartDateTime" : presentDateString,
         "EndDateTime" : tomorrowDateString
-    }, processResponse);
+    }, processClasses);
+}
+
+/**
+ * Using the class retrieved by API call, adds emails of 
+ * attendees into an array.
+ */
+async function processEnrollments(err, data){
```

Comment:
This function name is kind of confusing because the name is way too generic. This is a problem with writing call back function outside of the actual functions themselves because usually what these function will do are very similar to their original functions. Since this is a small function, you can just declare this entire callback function inside the function themselves. 

Or, if you want to keep it this way, this function will be easier to understand if it is named `_getClientEmails`, so that we can distinguish callback functions vs actual functions, and now thinking about this, to keep the functionalities separated, let's just return the list of emails with this function. and we can make the functionalities of how to use the client emails in a different place. Also, if you check the monday board task, all it said was to get the emails from the class. Someone else can use your function later.